strr-web/components/bcros/form-section/property/Form.vue

```diff
@@ -273,6 +303,80 @@ const validateNumberOfRoomsForRent = () => {
   }
 }
 
+const streetNumberError = ref('')
+const streetNameError = ref('')
+const unitNumberError = ref('')
+const addressLineTwoError = ref('')
+const cityError = ref('')
+const provinceError = ref('')
+const postalCodeError = ref('')
+
+const validateStreetNumber = () => {
+  const parsed = propertyDetailsSchema.safeParse(formState.propertyDetails).error?.errors
+  const error = parsed?.find(error => error.path.includes('streetNumber'))
+  streetNumberError.value = error ? error.message : ''
+}
+
+const validateStreetName = () => {
+  const parsed = propertyDetailsSchema.safeParse(formState.propertyDetails).error?.errors
+  const error = parsed?.find(error => error.path.includes('streetName'))
+  streetNameError.value = error ? error.message : ''
+}
+
+const validateUnitNumber = () => {
+  const parsed = propertyDetailsSchema.safeParse(formState.propertyDetails).error?.errors
+  const error = parsed?.find(error => error.path.includes('unitNumber'))
+  unitNumberError.value = error ? error.message : ''
+}
+
+const validateAddressLineTwo = () => {
+  const parsed = propertyDetailsSchema.safeParse(formState.propertyDetails).error?.errors
+  const error = parsed?.find(error => error.path.includes('addressLineTwo'))
+  addressLineTwoError.value = error ? error.message : ''
+}
+
+const validateCity = () => {
+  const parsed = propertyDetailsSchema.safeParse(formState.propertyDetails).error?.errors
+  const error = parsed?.find(error => error.path.includes('city'))
+  cityError.value = error ? error.message : ''
+}
+
+const validateProvince = () => {
+  const parsed = propertyDetailsSchema.safeParse(formState.propertyDetails).error?.errors
+  const error = parsed?.find(error => error.path.includes('province'))
+  provinceError.value = error ? error.message : ''
+}
+
+const validatePostalCode = () => {
+  const parsed = propertyDetailsSchema.safeParse(formState.propertyDetails).error?.errors
+  const error = parsed?.find(error => error.path.includes('postalCode'))
+  postalCodeError.value = error ? error.message : ''
+}
+
+const resetFieldError = (errorTypes: string[]) => {
```

Comment:
Ok I see it now... can reset multiple fields...

Reply:
Yes, with `streetNumber` and `streetName` [which triggers address complete], I want to reset errors for `city`, `postal code`, etc.